index.js

```diff
@@ -24,6 +24,11 @@ exports.run = function(runner, specs) {
       debug('cucumber command: "' + cliArguments.join(' ') + '"');
 
       Cucumber.Cli(cliArguments).run(function (isSuccessful) {
+        if (!isSuccessful) {
+          reject(new Error('Cucumber has fail.'))
+          return
```

Comment:
Good point too! But is `onComplete` must be called when test fails ?
Protractor doc says: `must be called when tests are finished.`... So... Yes I think.
I adapted the code for calling `onComplete` before checking `isSuccessful`.
Thanks